gui/slint/wizard.slint

```diff
@@ -0,0 +1,248 @@
+
+import { Button, LineEdit } from "std-widgets.slint";
+import { GlobalCallbacks } from "globals.slint";
+
+enum WizardPage {
+    Intro,
+    System,
+    Game,
+    Firmware,
+    Conclusion
+}
+
+export enum FileValidationResult {
+    Ok,
+    NotAbsolutePath,
+    DoesNotExist,
+    NotDirectory,
+    NotFile,
+    SameAsSystemDir
+}
+
+export component Wizard inherits Window {
+    min-width: 1000px;
+    min-height: 500px;
+
+    pure callback validate_system_dir(string) -> FileValidationResult;
+    pure callback validate_games_dir(string, string) -> FileValidationResult;
```

Comment:
We should not need this since the game management will be handled by the PS4 system software.

Reply:
Wait, I guess we need something like this. The idea is the PS4 should try to mount some RW partitions as a storage to install the game and other data. But let's remove this for now until we know exactly how many RW partition the PS4 will try to mount.

Reply:
Just removed it.